### ISSUE TITLE
Just a little request for an additional notebook

### ISSUE THREAD

**Issue by ivan-marroquin** [thilowrona/seismic_deep_learning#1]
Hi there!

Many thanks for making available these tutorials. I was hoping that you could add another notebook showing how to import seismic data, scale amplitude range, and same inlines (or crosslines) to a tiff file.

Kind regards,

Ivan

**Comment by thilowrona** [comment]
Hi Ivan,

thanks a lot for your message! Loading seismic data (usually stored in .segy format) can be a bit tricky, because information, such as number of traces, inlines and crosslines, are not always stored in the same way, so you probably need to explore your seismic files a bit. 

The best tools I've found for loading seismic data in Python are [segyio](https://github.com/equinor/segyio) and [segpy](https://github.com/sixty-north/segpy). They have plenty of examples for 2-D and 3-D data. There is also an example for 3-D data in [Tutorial-3](https://github.com/thilowrona/seismic_deep_learning/blob/master/tutorial-3/tutorial-3.ipynb) In [6]. This also shows how to scale your data. 

Finally, I can recommend Pythontool-pro by Cegal, which allows you to transfer all kinds of data (seismics, interpretations) between Python and Petrel - a great tool for machine learning on seimsic data.

To save seismic data as tiff, I have converted it from an array (NumPy) to an image ([Pillow](https://pillow.readthedocs.io/en/stable/)) and then saved it.

Hope any of these are useful!

All the best,
Thilo
